src/BUILD

```diff
@@ -216,7 +216,7 @@ cc_library(
         "@opencv//:opencv",
     ],
     local_defines = [
-        "SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_DEBUG"
```

Comment:
We need to ensure that performance does not drop with log level INFO and this build flag enabled